### DISCUSSION TITLE
Thank you for https://github.com/morrownr/8814au 8814

### DISCUSSION THREAD

**Discussion by a-spicer** [morrownr/8814au#121] [General]
I am on a new install of Lubuntu 23.10. I had use my Edimax EW-7833UAC on previous version of Lubuntu (20.10 I believe) So I'm at it again with a new Kernel version 6.5.0. Other web pages / git downloads were failing to install. I decided to Google Search more specifically for version 6.5 and found your git repository. Thank You Thank You Thank You. It works. Your shell scripts to automate it made it very easy to install.

**Comment by morrownr** [comment]
Hi @a-spicer 

> Other web pages / git downloads were failing to install.

This may be the last maintained 8814au repo. I'm not aware of any others. This is not an easy driver to maintain but I'll try to keep it going as long as I can.

> Thank You. It works. 

Glad it worked.

There is much more here than this driver. The Main Menu for this site is:

https://github.com/morrownr/USB-WiFi

You might be interested in the information.

Cheers,

@morrownr

**Comment by a-spicer** [comment]
Yes, thanks, I saw that other stuff ... and was considering buying another USB WiFi or two. I need to move up to 802.11ax, for home and work places. We work on embedded products (routers and WiFi client devices) and I bought 2 of these 7833UAC to be able to Wireshark capture WiFi traffic.

Let me know if I can send you a coffee or a beer, somehow.

---
Alan Spicer 

Oakland Park, FL.

Tel. +1 305-926-5578
Email: ***@***.***, ***@***.***

**Comment by morrownr** [comment]
> Let me know if I can send you a coffee or a beer, somehow.

The offer is appreciated but your thank you is more than enough.

If you need advise on a good AX adapter, let me know. The Plug and Play list can't cover everything.

FYI: You might buy one mt7921au based adapter now and pick up another adapter at some point in 2024. The reason I say this is because Mediatek just merged their mt7925 driver into the 6.7 kernel. The my7925 is WiFi 7 and the chipset is for USB and PCIe so we should see new adapters that are WiFi 7 at some point in 2024.
